spec/lib/tasks/backfill_sponsor_id_rake_spec.rb

```diff
@@ -21,21 +21,16 @@
     end
   end
 
-  let(:pending_enrollment) { create(:in_person_enrollment, :pending, :with_nil_sponsor_id) }
-  let(:expired_enrollment) { create(:in_person_enrollment, :expired, :with_nil_sponsor_id) }
-  let(:failed_enrollment) { create(:in_person_enrollment, :failed, :with_nil_sponsor_id) }
+  let(:pending_enrollment) { create(:in_person_enrollment, :pending) }
+  let(:expired_enrollment) { create(:in_person_enrollment, :expired) }
+  let(:failed_enrollment) { create(:in_person_enrollment, :failed) }
   let(:enrollment_with_service_provider) do
-    create(:in_person_enrollment, :with_service_provider, :with_nil_sponsor_id)
```

Comment:
To ensure this factorybot trait doesn't linger in the codebase, I made a cleanup ticket, [LG-14115](https://cm-jira.usa.gov/browse/LG-14115).  We can pick up this ticket after the EIPP friends and family launch.

Reply:
Oh good, I wrote a note to myself to handle this after launch too. Ticket is better